Jenkinsfile

```diff
@@ -58,7 +50,16 @@ pipeline {
         stage('Build') {
             steps {
                 container('node') {
-                    buildInstaller()
+                  sh "rm -rf ${YARN_CACHE_FOLDER}"
+                  sh "yarn --ignore-engines --unsafe-perm"
+                }
+            }
+        }
+
+        stage('Codechecks ESLint') {
+            steps {
+                container('node') {
+                    sh "yarn lint -o eslint.xml -f checkstyle"
```

Comment:
Should a `**/eslint.xml` file be added to `.gitignore`? After running this command locally, I got three XML files that git wants me to add.

Reply:
Good catch, thanks @cdamus! I added it to `.gitignore` with the updated commit.